process/transaction/shardProcess.go

```diff
@@ -334,6 +340,7 @@ func (txProc *txProcessor) processTxFee(
 		return big.NewInt(0), nil
 	}
 
+	//TODO: just for crossShardSCCall have a different cost? What about builtInFunctions and Deploys?
```

Comment:
deploy is always intra shard. built in function cost is always taken in self shard.